nucleus-core/src/main/java/io/github/nucleuspowered/nucleus/modules/fly/commands/FlyCommand.java

```diff
@@ -58,7 +58,7 @@
             context.sendMessage(fly ? "command.fly.player.on" : "command.fly.player.off", player.getName());
         }
 
-        context.sendMessage(fly ? "command.fly.on" : "command.fly.off");
+        player.sendMessage(context.getMessage(fly ? "command.fly.on" : "command.fly.off"));
```

Comment:
Thanks for looking at this. What you've done here may be an issue though as (in theory) you've sent a message to player, but using the command invoker's locale instead. This should use `context.sendMessageTo(player, ...)`

```suggestion
        context.sendMessageTo(player, fly ? "command.fly.on" : "command.fly.off"));
```

Reply:
Ah I see. Thank you for pointing this out. Commited the suggestion to my fork.